package.json

```diff
@@ -63,6 +63,7 @@
   },
   "scripts": {
     "lint": "eslint --ext .d.ts,.ts,.test-d.ts,.cjs . && prettier --check ./src/main/compendiums/**/*.json ./src/main/lang/*.json",
+    "lint-fix": "eslint --ext .d.ts,.ts,.test-d.ts,.cjs . --fix && prettier --check ./src/main/compendiums/**/*.json ./src/main/lang/*.json",
```

Comment:
Are you aware that we have auto fix on commit? No problem running by hand, just making you aware.

Reply:
How do I get that to run? Is it a flag I pass to `git commit`?

Reply:
Not it happens automatically through a git hook. I think you have to run `npm install` or something like that once on start.

Reply:
Then I don't know what I did wrong in a previous commit, because I've had the post-push hooks on github complain to me about style when I pushed one of my commits

Reply:
One thing to change here, `prettier` should have the `--write` option to also make it fix things.